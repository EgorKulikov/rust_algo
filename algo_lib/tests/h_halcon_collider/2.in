3
1 2 1 1
1 3 2 4
4 3 5 2
