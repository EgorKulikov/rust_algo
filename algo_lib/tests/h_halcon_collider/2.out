2.8284271247461903
1
TL
6.708203932499369
3
BR
64.62197768561404
21
BR
